applications/quantumleap/chart/Chart.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v2
+name: quantumleap
+description: A Helm chart for Kubernetes to deploy QuantumLeap using the official Helm Chart.
+type: application
+version: 0.1.0
+appVersion: "0.8.0"
+keywords:
+  - quantumleap
+  - helm
+  - chart
+sources:
+  - https://github.com/FIWARE-Ops/marinera/applications/quantumleap
+dependencies:
+- name: quantumleap
+  version: 0.1.18
+  repository: https://orchestracities.github.io/charts/
+maintainers:
```

Comment:
Very complete Chart.yaml, my main concern is that we (you Ernesto and Arturo) appear as mantainers...while this is just an umbrella chart consuming the official quantum leap one as a dependency...I mean, not much we are maintaining, we are just consuming. If I were someone new to this, I would think you are the maintainers of the orchestracities chart...

Reply:
We could remove the maintainer part altogether. Makes sense what you say